programmer-humour/script.js

```diff
@@ -0,0 +1,37 @@
+const api = "https://xkcd.now.sh/?comic=latest"
+let lastComic
+
+document.addEventListener('DOMContentLoaded', function () {
+  fetchComic(api);
+});
+
+function fetchComic(api) {
+  fetch(api)
+    .then((response) => response.json())
+    .then((data) => lastComic = data)
+    .then(() => renderPage(lastComic))
+    .catch((error) => console.log(error));
+}
+
+function renderPage(data) {
+  console.log(data);
+  const root = document.querySelector("#root")
+  const html = htmlGenerator(data.title, data.img, data.alt);
+
+  root.innerHTML = html;
+}
+
+function htmlGenerator(title, imageUrl, imageAlt) {
+  let html = `<h2>${title}</h2>
+    <img src="${imageUrl}" alt="${imageAlt}">
```

Comment:
Does this work? Do you need `` ` `` instead of `"`?

Maybe I'm missing something, let me know if I am 😄

Reply:
Maybe I am doing something wrong. But, when I change ``` to ", the page stops working, and VS Code starts showing red marks under some parts of the code. Like this:

![image](https://github.com/CodeYourFuture/Module-JS3/assets/16962159/2e1cd471-8cfd-437c-a3ac-1f0af0686cf8)

Reply:
Thanks, mate!